.gitignore

```diff
@@ -98,3 +98,4 @@ public/system/
 
 ## Sitemap
 public/sitemap.xml
+.byebug_history
```

Comment:
@awead I'm curios why it is needed now.  Should it be part of a global git ignore instead of and individual project?